modules/tinymce/src/core/test/ts/browser/fmt/ListIndentationFormatTest.ts

```diff
@@ -0,0 +1,32 @@
+import { Keys } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyContentActions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.fmt.ListItemFormatTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    apply_list_formatting: false,
+    base_url: '/project/tinymce/js/tinymce',
+    plugins: 'lists'
+  }, [], true);
+
+  context('Apply list formatting', () => {
+
+    it('TINY-8961: Apply multiple formats sequentially to entire list', () => {
```

Comment:
Maybe we can inspect the computed styles with Css.get on the LI elements to see if this works correctly?